tailcall-fixtures/fixtures/generator/simple-json.json

```diff
@@ -1,21 +1,26 @@
 {
   "inputs": [
     {
-      "URL": {
-        "url": "https://jsonplaceholder.typicode.com/posts/1"
+      "curl": {
+        "src": "https://jsonplaceholder.typicode.com/posts/1",
+        "headers": {
+          "Content-Type": "application/json",
+          "Authorization": "Bearer your_access_token_here",
```

Comment:
security data should not be stored in files and usually should be provided by env or similar. For headers we should support templates to allow specify data from env